server/tracedb/jaegerdb.go

```diff
@@ -1,25 +1,24 @@
-package jaegerdb
+package tracedb
 
 import (
 	"context"
 	"fmt"
 	"io"
 
-	pb "github.com/kubeshop/tracetest/server/go/internal/proto-gen-go/api_v3"
-	"github.com/kubeshop/tracetest/server/go/tracedb"
+	pb "github.com/kubeshop/tracetest/internal/proto-gen-go/api_v3"
 	"go.opentelemetry.io/collector/component/componenttest"
 	"go.opentelemetry.io/collector/config/configgrpc"
 	v1 "go.opentelemetry.io/proto/otlp/trace/v1"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/status"
 )
 
-type JaegerTraceDB struct {
+type jaegerTraceDB struct {
 	conn  *grpc.ClientConn
 	query pb.QueryServiceClient
 }
 
-func New(config *configgrpc.GRPCClientSettings) (tracedb.TraceDB, error) {
+func newJaegerDB(config *configgrpc.GRPCClientSettings) (*jaegerTraceDB, error) {
```

Comment:
any reason for returning a `*jaegerTraceDB` instead of `TraceDB`? Just like `TempoTraceDB` is doing.

Reply:
Nope, just forgot to update it